static/test/javascripts/spec/common/search-tool.spec.js

```diff
@@ -0,0 +1,197 @@
+define([
+    'bean',
+    'bonzo',
+    'common/utils/$',
+    'common/modules/search-tool'
+], function (
+    bean,
+    bonzo,
+    $,
+    SearchTool
+    ) {
+
+    describe('Search tool', function () {
+        var container,
+            sut;
+
+        beforeEach(function () {
+            container = bonzo.create(
+                    '<div class="search-tool js-search-tool">' +
+                    '<div class="search-tool__form u-cf">' +
+                    '<input class="search-tool__input js-search-tool-input" type="text" data-link-name="weather-list-location" />' +
+                    '<button class="search-tool__btn"><i class="i i-search-white-36"></i></button>' +
+                    '</div>' +
+                    '<ul class="u-unstyled js-search-tool-list"></ul>' +
+                    '</div>'
```

Comment:
Just realised this is a test spec, so you could just use the `fixtures` helper instead.